### PR TITLE
ANY23-444 Update all dependencies and plugins

### PR DESCRIPTION
**Pull request by lewismc**
This PR addresses https://issues.apache.org/jira/browse/ANY23-444

In it's current state the PR introduces at least one test failure as follows
```
[ERROR] Failures:
[ERROR]   EmbeddedJSONLDExtractorTest.testJSONLDUnescapedCharacters:76->AbstractExtractorTestCase.assertExtract:227->AbstractExtractorTestCase.assertExtract:212->AbstractExtractorTestCase.assertNoIssues:386 Unexpected issue for extractor html-embedded-jsonld : [FATAL: 	'Unrecognized character escape (CTRL-CHAR, code 8)' 	(5,31), FATAL: 	'Illegal unquoted character ((CTRL-CHAR, code 10)): has to be escaped using backslash to be included in string value' 	(6,43), FATAL: 	'Illegal unquoted character ((CTRL-CHAR, code 10)): has to be escaped using backslash to be included in string value' 	(6,22)]
[INFO]
[ERROR] Tests run: 449, Failures: 1, Errors: 0, Skipped: 0
```
I spent some time looking into the error but couldn't resolve it.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -270,17 +270,17 @@
...
 
-    <httpclient.version>4.5.7</httpclient.version>
-    <httpcore.version>4.4.11</httpcore.version>
+    <httpclient.version>4.5.10</httpclient.version>
+    <httpcore.version>4.4.12</httpcore.version>
     <owlapi.version>5.1.11</owlapi.version>
-    <poi.version>4.0.1</poi.version>
-    <rdf4j.version>2.4.4</rdf4j.version>
+    <poi.version>4.1.0</poi.version>
+    <rdf4j.version>3.0.0</rdf4j.version>
```
Comment (HansBrende) [inline pom.xml]: Upgrading rdf4j is what caused your bug. Can we hold off on that? Upgrading rdf4j is worthy of being solved in its own separate issue. There are some hacks in the Jsonld extractor I need to fix at the same time the upgrade happens. Better to leave this for ANY23-433.

File: service/pom.xml [hunk 2]
```diff
@@ -33,7 +33,7 @@
 
   <properties>
     <!-- the following property is used in the bash script as well, don't remove it! -->
-    <jetty.runner.version>9.4.14.v20181114</jetty.runner.version>
+    <jetty.runner.version>10.0.0-alpha0</jetty.runner.version>
```
Comment (HansBrende) [inline service/pom.xml]: The latest stable version is `9.4.20.v20190813`, can we use that instead of the pre-release version?

File: pom.xml (hunk 1 above)
Comment (lewismc) [reply pom.xml]: Thanks @HansBrende I am happy to delay this.

**Comment by lewismc** [comment]
Thanks for review @HansBrende I've addressed everything you mentioned. 
Using the following environment I get test failures
```
Apache Maven 3.6.0 (97c98ec64a1fdfee7767ce5ffb20918da4f719f3; 2018-10-24T11:41:47-07:00)
Maven home: /usr/local/Cellar/maven/3.6.0/libexec
Java version: 13, vendor: Oracle Corporation, runtime: /Library/Java/JavaVirtualMachines/jdk-13.jdk/Contents/Home
Default locale: en_US, platform encoding: UTF-8
OS name: "mac os x", version: "10.14.6", arch: "x86_64", family: "mac"
```
**mvn clean test**
```
[INFO] Running org.apache.any23.extractor.rdfa.RDFa11ParserTest
[ERROR] Tests run: 12, Failures: 0, Errors: 2, Skipped: 0, Time elapsed: 0.142 s <<< FAILURE! - in org.apache.any23.extractor.rdfa.RDFa11ParserTest
[ERROR] testIsRelativeNegative(org.apache.any23.extractor.rdfa.RDFa11ParserTest)  Time elapsed: 0.072 s  <<< ERROR!
org.mockito.exceptions.base.MockitoException:

Mockito cannot mock this class: interface org.w3c.dom.Document.

Mockito can only mock non-private & non-final classes.
If you're not sure why you're getting this error, please report to the mailing list.


Java               : 13
JVM vendor name    : Oracle Corporation
JVM vendor version : 13+33
JVM name           : Java HotSpot(TM) 64-Bit Server VM
JVM version        : 13+33
JVM info           : mixed mode, sharing
OS name            : Mac OS X
OS version         : 10.14.6


Underlying exception : java.lang.IllegalArgumentException: org.mockito.codegenDocument$MockitoMock$1538161731 must be defined in the same package as org.mockito.codegen.InjectionBase/package
	at org.apache.any23.extractor.rdfa.RDFa11ParserTest.testIsRelativeNegative(RDFa11ParserTest.java:86)
Caused by: java.lang.IllegalArgumentException: org.mockito.codegenDocument$MockitoMock$1538161731 must be defined in the same package as org.mockito.codegen.InjectionBase/package
	at org.apache.any23.extractor.rdfa.RDFa11ParserTest.testIsRelativeNegative(RDFa11ParserTest.java:86)

[ERROR] testGetDocumentBase(org.apache.any23.extractor.rdfa.RDFa11ParserTest)  Time elapsed: 0.03 s  <<< ERROR!
org.mockito.exceptions.base.MockitoException:

Mockito cannot mock this class: interface org.w3c.dom.Document.

Mockito can only mock non-private & non-final classes.
If you're not sure why you're getting this error, please report to the mailing list.


Java               : 13
JVM vendor name    : Oracle Corporation
JVM vendor version : 13+33
JVM name           : Java HotSpot(TM) 64-Bit Server VM
JVM version        : 13+33
JVM info           : mixed mode, sharing
OS name            : Mac OS X
OS version         : 10.14.6


Underlying exception : java.lang.IllegalArgumentException: org.mockito.codegenDocument$MockitoMock$1341603674 must be defined in the same package as org.mockito.codegen.InjectionBase/package
	at org.apache.any23.extractor.rdfa.RDFa11ParserTest.testGetDocumentBase(RDFa11ParserTest.java:48)
Caused by: java.lang.IllegalArgumentException: org.mockito.codegenDocument$MockitoMock$1341603674 must be defined in the same package as org.mockito.codegen.InjectionBase/package
	at org.apache.any23.extractor.rdfa.RDFa11ParserTest.testGetDocumentBase(RDFa11ParserTest.java:48)
```

**Comment by HansBrende** [comment]
@lewismc I bet the problem is with mockito. Try upgrading mockito-core to version 2.28.2.

**Comment by lewismc** [comment]
@HansBrende I fixed it... I'll submit my update ASAP

**Comment by lewismc** [comment]
> What's the difference in semantics between

https://stackoverflow.com/questions/32604733/error-os-windows-and-the-assembly-descriptor-contains-a-nix-specific-root-relat

I'll merge now, thanks for review @HansBrende

**Pull request merged**